python/llm/example/CPU/PyTorch-Models/Model/Qwen-VL-Chat/README.md

```diff
@@ -0,0 +1,74 @@
+# Qwen-VL-Chat
```

Comment:
Maybe we should change the folder name to qwen-vl.

Same other places: Qwen-VL-Chat model -> Qwen-VL model

Reply:
Also maybe `SimSun.tff` and `Session_2.png` should not be added here in the qwen-vl folder